tools/build_defs/framework.bzl

```
@@ -324,6 +324,10 @@ def cc_external_rule_impl(ctx, attrs):
         else:
             make_commands.append(line)
 
+    ld_library_path = _define_ld_library_path(attrs.deps)
+    if len(ld_library_path) > 0:
+        define_variables.append("export LD_LIBRARY_PATH=\"{}\"".format(":".join(ld_library_path)))
+
```

This needs to add a default of appending to LD_LIBRARY_PATH that may already be set via `action_env` or via `cc_env` from the toolchain.

Something like:
```suggestion
    ld_library_path = _define_ld_library_path(attrs.deps)
    if len(ld_library_path) > 0:
        define_variables.append("export LD_LIBRARY_PATH=\"{}:$${LD_LIBRARY_PATH:-}$$\"".format(":".join(ld_library_path)))

```

---

Sorry for ghosting out on this!  I'll try to carve out some time this week or next to freshen things up.